### ISSUE TITLE
Support v3.0

### ISSUE THREAD

**Issue by vitorenesduarte** [Ecwid/ecwid-mailchimp#31]
Since the others versions are deprecated, there will be support for v3.0?

**Comment by edbras** [comment]
Can somebody of the dev team please indicate what changes need to be made to support v3.0?
As such we know where to start and upgrade the lib.
Please let me/use know?
- Ed

**Comment by bloo** [comment]
@basiliscus is this project dead? we need to upgrade to 3.0 support so wondering if we use this client or find/write another. fyi @scotdalton @luksow @jglodek

**Comment by basiliscus** [comment]
@bloo this project is more dead than alive. My apologies. I'm not going to spend time on it unless the company I work for decides that we need v3.0 support

**Comment by edbras** [comment]
My advice: just use it and make changes yourself to support 3.0. That's what I did (for the parts that I needed), it's very straight forward.

**Comment by micheal-swiggs** [comment]
@edbras Do you have a visible project with your 3.0 changes? I will be using mailchimp and will want to use version 3.0.

**Comment by edbras** [comment]
[edbras-ited-mailchimp-0b8244ed5e73.zip](https://github.com/Ecwid/ecwid-mailchimp/files/79639/edbras-ited-mailchimp-0b8244ed5e73.zip)

This is een repo dump of the project on bitbucket.

**Comment by micheal-swiggs** [comment]
@edbras Thanks very much I'll have a look at this now.

**Comment by micheal-swiggs** [comment]
@edbras Is there any licence on this code? I was wondering if it would be ok to open source this? I noticed that there are dependencies your internal libraries. I'm currently trying to see if it is possible to remove these.

**Comment by edbras** [comment]
@micheal-swiggs : no, no licence. Go ahead and use it how you want it.  I was planning to open-source some stuff for a long time, but never got to it :(... How can I PM you the dependencies ? So you can copy the dependent code..

**Comment by micheal-swiggs** [comment]
@edbras That sounds great :) my email is swiggsmi87@hotmail.com .

**Comment by edbras** [comment]
I mailed it to you.

**Comment by marlowBlackwood** [comment]
@edbras @micheal-swiggs did y'all end up going any further with the 3.0 update? I need to use mailchimp with java as well for a current project and would share back any new API methods I add.

**Comment by edbras** [comment]
@marlowBlackwood: no

**Comment by NetoDevel** [comment]
@marlowBlackwood I am implementing a lib to consume the v3.0 version. It would be good contributors to help finish the lib.
Access and contribute: https://github.com/NetoDevel/mailchimp-java

**Comment by marlowBlackwood** [comment]
@NetoDevel What do you need help with? It's been a while since I was working on this and the client's project requirements changed a bit since then (didn't end up needing mailchimp). I'm still happy to help though b/c it's likely we'll revisit the mailchimp integration eventually.

**Comment by NetoDevel** [comment]
@marlowBlackwood  
help end the lib with all requests v3 , I do not have much time because of work , and it would also be my first contact with open-source contributors.

**Comment by basiliscus** [comment]
Hi guys,

I (the project maintainer) have created a replacement project to support MailChimp API v3.0: https://github.com/Ecwid/maleorang . This project (ecwid-mailchimp) is being deprecated.

**Comment by micheal-swiggs** [comment]
Hi all, 

Another version under active development : https://github.com/micheal-swiggs/mailchimp-java. Any contributions welcome.
